src/client/components/modals/Organization.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react'
+import { Field, Text, Button } from '@aragon/ui'
+import PropTypes from 'prop-types'
+import { ModalWrapper, DisplayErrors } from './ModalWrapper'
+import { TextInputWithValidation } from '../styled-components'
+import { validateDAOAddress } from '../../utils/validation'
+
+const Organization = ({
+  ethereumAddress,
+  getFormValue,
+  onChange,
+  saveProfile,
+  organizationId,
+  savingError,
+}) => {
+  const [validationErrors, setValidationErrors] = useState({})
+
+  const validateAndSave = () => {
+    const errors = {}
+    if (
+      !validateDAOAddress(
+        getFormValue('organizations', organizationId, 'address')
+      )
+    )
+      errors['organization'] = 'Please provide valid DAO address'
```

Comment:
I'll defer to `prettier` if we have it, but if not, it looks really strange to my eyes to see this line hanging out at this indentation level by itself, after the awkward-to-read `if` statement. Maybe we can rewrite this as:

```
if (!validateDAOAddress(
  getFormValue('organizations', organizationId, 'address')
)) {
  errors['organization'] = 'Please provide valid DAO address'
}